enterprise/frontend/src/metabase-enterprise/clean_up/CleanupCollectionModal/CleanupCollectionModalFilters.tsx

```diff
@@ -0,0 +1,58 @@
+import { c, t } from "ttag";
+import _ from "underscore";
+
+import { Icon, Select, Flex, Text, Switch } from "metabase/ui";
+
+import { dateFilterOptions, isDateFilter, type DateFilter } from "./utils";
+
+interface CleanupCollectionModalFiltersProps {
+  dateFilter: DateFilter;
+  recursiveFilter: boolean;
+  onDateFilterChange: (dateFilter: DateFilter) => void;
+  onRecursiveFilterChange: (recursiveFilter: boolean) => void;
+}
+
+export const CleanupCollectionModalFilters = ({
+  dateFilter,
+  recursiveFilter,
+  onDateFilterChange,
+  onRecursiveFilterChange,
+}: CleanupCollectionModalFiltersProps) => (
+  <Flex
+    mt="1.5rem"
+    mb="1rem"
+    w="100%"
+    justify="space-between"
+    direction={{ base: "column", md: "row" }}
+    align={{ base: "start", md: "center" }}
+    gap={{ base: "md", md: "none" }}
+  >
+    <Flex
+      gap="sm"
+      align={{ base: "start", md: "center" }}
+      direction={{ base: "column", md: "row" }}
+    >
+      <Text fw="bold">{c(
+        `Prefixes a time span, reads as "Not viewed in over 6 months"`,
+      ).t`Not viewed in over`}</Text>
+      <Select
+        icon={<Icon name="calendar" />}
+        id="not-view-in-filter"
```

Comment:
Do we have a policy of prefixing ids (maybe with `mb-`) to avoid id collisions when embedded? Maybe we should?

Reply:
don't think i needed this, but good thought